src/wp-includes/media.php

```diff
@@ -1070,17 +1070,18 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 		list( $src, $width, $height ) = $image;
 
 		$attachment = get_post( $attachment_id );
-		$hwstring   = image_hwstring( $width, $height );
 		$size_class = $size;
 
 		if ( is_array( $size_class ) ) {
 			$size_class = implode( 'x', $size_class );
 		}
 
 		$default_attr = array(
-			'src'   => $src,
-			'class' => "attachment-$size_class size-$size_class",
-			'alt'   => trim( strip_tags( get_post_meta( $attachment_id, '_wp_attachment_image_alt', true ) ) ),
+			'src'    => $src,
+			'class'  => "attachment-$size_class size-$size_class",
+			'alt'    => trim( strip_tags( get_post_meta( $attachment_id, '_wp_attachment_image_alt', true ) ) ),
+			'width'  => $width,
+			'height' => $height,
```

Comment:
I don't think we should merge `width` and `height` into `$default_attr`. Doing that would allow overriding their values via `$attr`, which I don't think we should allow without further consideration.

I think the better solution would be to leave `$default_attr` untouched, and instead only add the two values to `$attr` _after_ the `wp_parse_args( $attr, $default_attr )` call.